jsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "ES6",
```

Comment:
I think this would need to be set to `esnext` to match what is in the `.eslintrc` file: https://github.com/rugk/unicodify/blob/fe9ecd12398aa52e12698b7b0039d263c8b6c2b6/.eslintrc#L4
See [here](https://code.visualstudio.com/docs/languages/jsconfig#_jsconfig-options) for more information.

Reply:
OKay yeah… or wait no it's `es2020` actually. [This corresponds to v12](https://en.wikipedia.org/wiki/ECMAScript#12th_Edition_%E2%80%93_ECMAScript_2021).

Reply:
If you look at your link, it would actually be `es2021`:
```
The 12th edition, or ECMAScript 2021, was published in June 2021.
```
That does not seem to be supported by the `jsconfig.json` file, which is why I suggested `esnext`. Anyway, `es2020` is probably fine, as I believe we are only using a couple features in ECMAScript 2021.